runtime/oti/UnsafeAPI.hpp

```diff
@@ -376,9 +376,9 @@ class VM_UnsafeAPI
 		return (I_32)arrayBase(currentThread);
 	}
 
-	static VMINLINE I_32 arrayIndexScale(J9ArrayClass *arrayClass)
```

Comment:
leave the casts as it is, it maks it clear that this API is just for arrays

Reply:
Fixed in 213aa82